docs/DeveloperGuide.adoc

```diff
@@ -852,6 +852,25 @@ _{More to be added}_
 
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
+[discrete]
+=== Use case: View contents of a specific week
+
+*MSS*
+
+1.  User requests to view a specific week
+2.  AddressBook shows content of the specified week
```

Comment:
shall we change the name to ProgressChecker?

Reply:
Ok, will amend.